wagtail_localize/wagtail_hooks.py

```diff
@@ -415,3 +416,21 @@ def register_wagtail_localize2_report_menu_item():
         icon_name="site",
         order=9000,
     )
+
+
+@hooks.register("register_log_actions")
+def wagtail_localize_log_actions(actions):
+    @actions.register_action("wagtail_localize.convert_to_alias")
+    class ConvertToAliasActionFormatter(LogFormatter):
+        label = _("Convert page to alias")
```

Comment:
We should use ``gettext_lazy`` here since the ``register_log_actions`` hook will be cached and reused for multiple users. Keeping the two below as `_()` should be fine though